src/views/Timesheets/index.js

```diff
@@ -418,7 +418,7 @@ const Timesheets = (props) => {
                         value={selectedDateIn}
                         onChange={date => {
                           let dateToChange = date;
-                          handleDateChange1(dateToChange);
+                          handleDateChangeIn(dateToChange);
```

Comment:
Whoops! I said handle...2 instead of handle...1 in my earlier comment. The suggestion I meant to make was that 
`date = > { 
let dateToChange = data; 
handleDataChangeIn(dateToChange); }`

just be replaced with `handleDateChangeIn`. 
But am I missing something?

Reply:
Nevermind the above comment. I just saw the change 👍